tests/integration/rejectOnNotFound.test.ts

```diff
@@ -44,6 +44,7 @@ testIntegration({
   },
   prismaClient(prismaClientPackage) {
     // This global setting should have no effect on Nexus Prisma
+    // TODO:  prisma:warn `rejectOnNotFound` option is deprecated and will be removed in Prisma 5. Please use `prisma.user.undefined` method instead
```

Comment:
Can this TODO be moved to an issue rather than a comment in the code?

Reply:
+1 issue for this